presto-parquet/src/test/java/com/facebook/presto/parquet/writer/TestParquetWriter.java

```diff
@@ -77,6 +86,66 @@ public void testRowGroupFlushInterleavedColumnWriterFallbacks()
         }
     }
 
+    @Test
+    public void testMetadataCreation()
```

Comment:
@auden-woolfson can you help me understand how these tests run before and after your main code changes?